src/System.Linq.Expressions/tests/BinaryOperators/Assignment/OpAssign.cs

```diff
@@ -224,7 +224,7 @@ public void ThrowIfNoSuchBinaryOperation(MethodInfo assign)
             Assert.Throws<InvalidOperationException>(() => withAssignment(variable, value));
         }
 
-        private static IEnumerable<object[]> AssignmentMethods()
+        public static IEnumerable<object[]> AssignmentMethods()
```

Comment:
Yes, there's some strangeness here with XUnit. I first found that if I've made a silly mistake like a typo that means it can't find any such member, or the call is ambiguous, etc. then it complains about there being no public member, while it doesn't actually require the member to be public, which I put down to error messages not being sync with the code that uses it. However, other times it really does seem to insist on the member being public. I'd put that down to my having introduced an ambiguity that a public-first sort-order removed, but I didn't actually investigate to confirm that that was the case.